### ISSUE TITLE
Computation of MSE

### ISSUE THREAD

**Issue by haichaoyu** [tegg89/SRCNN-Tensorflow#9]
In model.py, line 56.   

    # Loss function (MSE)
    lf.loss = tf.reduce_mean(tf.reduce_sum(tf.square(self.labels - self.pred), reduction_indices=0))

It seems not the definition of MSE. How about 

    lf.loss = tf.reduce_mean(tf.square(self.labels - self.pred))

Thanks.

**Comment by tegg89** [comment]
@haichaoyu Thanks for valuable comment!
